editor-open.cabal

```diff
@@ -37,9 +37,13 @@ library
   build-depends:
      base ==4.8.*
    , bytestring
+   , conduit
```

Comment:
If you're going to use the `connect` and `fuse` functions like I suggested, the lower bound is `conduit >=1.2.3`